src/components/common/ClientIntakeForm/IntakeForms/Finances.js

```diff
@@ -1,175 +1,189 @@
 import React from 'react';
-import { Form, Input, Checkbox } from 'antd';
+import { Form, Checkbox, Input } from 'antd';
+import { useState } from 'react';
+
+const initialFormValues = {
+  eviction: false,
+  landlordDebt: false,
+  criminalHistory: false,
+  poorCredit: false,
+  opName: false,
+  SSI: false,
+  SSDI: false,
+  studentLoan: false,
+  medicalBills: false,
+  creditCard: false,
+  autoLoan: false,
+};
+
+const familyHistory = [
+  {
+    name: 'eviction',
+    label: 'Eviction',
+  },
+  {
+    name: 'landlordDebt',
+    label: 'Landlord Debt',
+  },
+  {
+    name: 'criminalHistory',
+    label: 'Criminal History',
+  },
+  {
+    name: 'poorCredit',
+    label: 'No Credit History',
+  },
+  {
+    name: 'noRentalHistory',
+    label: 'No Rental History',
+  },
+];
+
+const incomeSource = [
+  {
+    name: 'TANF',
+    label: 'TANF',
+  },
+  {
+    name: 'SSI',
+    label: 'SSI',
+  },
+  {
+    name: 'SSDI',
+    label: 'SSDI',
+  },
+  {
+    name: 'childSupport',
+    label: 'Child Support',
+  },
+];
+
+const typesOfDebt = [
+  {
+    name: 'studentLoan',
+    label: 'Student Loan',
+  },
+  {
+    name: 'creditCard',
+    label: 'Credit Card',
+  },
+  {
+    name: 'medicalBills',
+    label: 'Medical Bills',
+  },
+  {
+    name: 'autoLoan',
+    label: 'Auto Loan',
+  },
+];
 
 const Finances = () => {
   const [form] = Form.useForm();
+  const [formValues, setFormValues] = useState(initialFormValues);
+
+  const onChange = e => {
+    e.preventDefault();
+    setFormValues({ ...formValues, [e.target.name]: !e.target.value });
```

Comment:
Is the target value for this form always a boolean?

Reply:
yes, the boolean value relates to the boxes being clicked or not (true or false)